### ISSUE TITLE
get a different loss and accuracy curve

### ISSUE THREAD

**Issue by qimw** [www0wwwjs1/Matrix-Capsules-EM-Tensorflow#31]
Hi, and I am using the default parameters training the net, but I got some different loss and training accuracy curves compared with the README file. Has anyone meet this kind of things?
* smallNORB all loss:
![sn_all_loss](https://user-images.githubusercontent.com/18625505/39956876-81e02d74-561b-11e8-9534-c66f5a6868bb.png)
* smallNORB training accuracy:
![sn_train_acc](https://user-images.githubusercontent.com/18625505/39956914-38c6839e-561c-11e8-8ee1-96481d8eb027.png)
* mnist all loss:
![mn_all_loss2](https://user-images.githubusercontent.com/18625505/39956898-ea75d9e2-561b-11e8-81b4-8d3b521110d4.png)
* mnist training accuracy:
![mn_train_acc2](https://user-images.githubusercontent.com/18625505/39956902-f2bce96a-561b-11e8-977d-4062e515d09a.png)

**Comment by yhyu13** [comment]
@qimw @www0wwwjs1 

Your results looks pretty good. 

I apologize for the confusion, the learning curves we proposed include weight decay actually. However, by default, the weight decay loss is set to false ([here](https://github.com/www0wwwjs1/Matrix-Capsules-EM-Tensorflow/blob/0196eadc8dd38f476d3244d2ea577a1c7cdf6e8d/config.py#L20)). From your result, we can tell that weight decay is a essential regularization.

**Comment by qimw** [comment]
Hi, thank you for your prompt reply : )
